test-utils/fixtures/MovementSubmissionFailureFixtures.scala

```diff
@@ -45,6 +45,14 @@ trait MovementSubmissionFailureFixtures extends BaseFixtures {
     hasBeenFixed = false
   )
 
+  val destinationWarehouseExciseFailure: MovementSubmissionFailure = MovementSubmissionFailure(
+    errorType = ExciseIdForTaxWarehouseOfDestinationInvalidError.code,
+    errorReason = "The excise ID for the tax warehouse of destination is not valid",
+    errorLocation = Some("/IE815[1]/Body[1]/SubmittedDraftOfEADESAD[1]/EadEsadDraft[1]/LocalReferenceNumber[1]"),
```

Comment:
Probably worth making this None in places where it's not needed (i.e., all non-item errors)